tests/code/test_GetAllFacilitators.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from app.logic.getAllFacilitators import getAllFacilitators
+
+
+@pytest.mark.integration
+def test_getAllFacilitators():
+    userFacilitator = getAllFacilitators()

```

Comment:
since it is a getAll function, you should do a test where there are multiple facilitators, as well as a test where there are none

Reply:
We tested for multiple users by indexing the different facilitator objects in the list. We couldn't really think of a way to test for no facilitators especially after having tested for multiple facilitators. And removing all facilitators from the test_data.py would cause the website functionality to fail. So we just asserted that the length of the facilitators list was >=1 so it makes sure there is at least one facilitator.